__tests__/app.test.js

```diff
@@ -123,7 +123,46 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("GET 200 responds with an array of the articles with the requested topic.", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(1);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET 200 responds with an array of the articles with the requested topic.", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
```

Comment:
Are we testing for any different behaviours in this test? If it auto passed or both this and the previous test were written, both failed and then both passed it's an indicator of an unnecessary test 👍